nostr.test.ts

```diff
@@ -0,0 +1,17 @@
+import { assertEquals } from "https://deno.land/std@0.202.0/assert/assert_equals.ts";
+import { PrivateKey } from "./key.ts";
+import { sign_event_v2 } from "./nostr.ts";
+import { verify_event_v2 } from "./nostr.ts";
+
+Deno.test("verify event v2", async () => {
```

Comment:
Maybe it would be better to put this part of the test in `event.test.ts`. You can directly compare two versions of events.